domains/api/http/transport.go

```diff
@@ -82,9 +84,61 @@ func MakeHandler(svc domains.Service, authn authn.Authentication, mux *chi.Mux,
 			), "freeze_domain").ServeHTTP)
 			roleManagerHttp.EntityRoleMangerRouter(svc, d, r, opts)
 		})
+
+		r.Route("/{domainID}/invitations", func(r chi.Router) {
+			r.Use(api.AuthenticateMiddleware(authn, true))
+			r.Post("/", otelhttp.NewHandler(kithttp.NewServer(
+				sendInvitationEndpoint(svc),
+				decodeSendInvitationReq,
+				api.EncodeResponse,
+				opts...,
+			), "send_invitation").ServeHTTP)
+			r.Get("/", otelhttp.NewHandler(kithttp.NewServer(
+				listDomainInvitationsEndpoint(svc),
+				decodeListInvitationsReq,
+				api.EncodeResponse,
+				opts...,
+			), "list_domain_invitations").ServeHTTP)
+			r.Route("/{userID}", func(r chi.Router) {
+				r.Get("/", otelhttp.NewHandler(kithttp.NewServer(
+					viewInvitationEndpoint(svc),
+					decodeInvitationReq,
+					api.EncodeResponse,
+					opts...,
+				), "view_invitation").ServeHTTP)
+				r.Delete("/", otelhttp.NewHandler(kithttp.NewServer(
+					deleteInvitationEndpoint(svc),
+					decodeInvitationReq,
+					api.EncodeResponse,
+					opts...,
+				), "delete_invitation").ServeHTTP)
+			})
+		})
```

Comment:
@felixgateru 
Can we have like endpoints like 
- /domains/{domainID}/invtiations 
- /domains/{domainID}/invtiations/{userID}
- /invtiations 
- /invtiations/accept
- /invtiations/reject